### PR TITLE
Liotad autostart branch

### PR DESCRIPTION
**Pull request by ApurvaH**
Autostarts liota daemon in background.

### CODE REVIEW

File: scripts/autostartliota [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# Short-Description: Starts liotad.py in background at reboot.
+### END INIT INFO
+
+PATH=/sbin:/usr/sbin:/bin:/usr/bin
+
+do_start() {
+	sudo python /etc/liota/packages/liotad.py &
+}
+
+case "$1" in
```
Comment (KohliDev) [inline scripts/autostartliota]: How the re-loading of loaded packages is taken care, in case of restart scenario?

File: setup.py [hunk 2]
```diff
@@ -123,5 +123,7 @@
                  ['config/liota.conf', 'config/logging.json']),
                 (os.path.abspath(os.sep) + '/../etc/liota',
                  ['BSD_LICENSE.txt', 'BSD_NOTICE.txt']),
+                (os.path.abspath(os.sep) + '/../etc/init.d',
```
Comment (KohliDev) [inline setup.py]: Don`t copy forcefully in /etc/init.d, make it a manual step and document in README.md if user wants to use this functionality.

**Comment by ApurvaH** [comment]
Hi Vaibhav,

There is a text file: liota/packages/package_auto.txt where you have to list the packages that you want to be loaded everytime.

I implemented the same thing in different manner. All the packages that you load are automatically recorded and the list is then used to load them at restart.
https://github.com/vmware/liota/compare/master...ApurvaH:autostart_branch

Since this functionality has already been implemented I removed it from the pull request.

Thanks,
Apurva

File: scripts/autostartliota [hunk 3]
```diff
@@ -0,0 +1,32 @@
+#! /bin/sh
```
Comment (pmasrani) [inline scripts/autostartliota]: Add the copyright header.

File: README.md [hunk 4]
```diff
@@ -54,6 +54,50 @@ In general, liota can be installed with:
 
 It requires a Python 2.7 environment already installed.
 
+## Autostarting Liota Daemon
+For starting liotad.py in background automatically at reboot perform the following steps: 
+
+* Copy autostartliota script present in scripts folder to location:
```
Comment (KohliDev) [inline README.md]: What is the location of script folder ???
And I don`t see "scripts/autostartliota" included in setup.py in this PR to exist at "/etc/liota" location.

**Pull request merged**